docs/application/dotnet/index.md

```diff
@@ -4,90 +4,32 @@ Tizen .NET is an exciting new way to develop applications for the Tizen operatin
-
-- **Xamarin.Forms as a UI framework**
-
-  Xamarin.Forms is a cross-platform UI toolkit that allows you to efficiently create native user interface layouts that can be shared across iOS, Android&trade;, Windows Phone, and Universal Windows Platform applications.
-
-- **TizenFX API**
-
-  Tizen .NET supports C# APIs, which expose Tizen-specific features:
-
-  - **Tizen.Account** provides CRUD (Create, Read, Update, Delete) account management functionality and the OAuth Core RFC 6749 protocol.
-  - **Tizen.Applications** provides the Tizen application framework, including application state change events, inter-application messaging, and notification services.
-  - **Tizen.Common** provides predefined color names.
-  - **Tizen.Content** provides content management services, such as file and media downloading, storing and indexing audio and video content, and associating content types with helper applications.
-  - **Tizen.Location** manages geographical location services and geofencing.
-  - **Tizen.Maps** provides methods to create map-aware applications.
-  - **Tizen.Messaging** provides methods to receive push notifications.
-  - **Tizen.Multimedia** interacts with media services, including audio playback, recording, and device policy.
-  - **Tizen.Network** controls connectivity devices and retrieves network information.
-  - **Tizen.NUI** (Natural User Interface) is a toolkit for creating applications with a rich GUI. The applications can run on a range of Tizen devices.
-  - **Tizen.Security** provides access to secure storage for passwords, keys, certificates, and other sensitive data.
-  - **Tizen.Sensor** provides sensor types and sensor information.
-  - **Tizen.System** provides device-specific services, including status, system information and settings, feedback, and sensor control and access.
-  - **Tizen.Telephony** provides call, modem, network, and SIM information.
-  - **Tizen.Uix** provides methods to recognize speech, synthesize voice from text, and play synthesized sound data.
-  - **Tizen.Log** provides methods to print log messages to the Tizen logging system.
-  - **Tizen.Tracer** provides methods to write trace messages to the system trace buffer.
-
-<a id="release"></a>
-## Tizen .NET Release Summary
-
-The following table describes the latest changes, additions, and fixes for Tizen .NET.
-
-![Tizen .NET release summary](media/release_summary.png)
+- [3rd party libraries for Tizen .Net](tutorials/library/library-list.md)
 
+  Introduces several 3rd party libraries that you can use to develop .NET applications. It is classified according to use and it can be selected according to the purpose of use.
 
 ### Visual Studio Tools for Tizen [![Download](media/ic_docs_download.png)](https://marketplace.visualstudio.com/items?itemName=tizen.VisualStudioToolsforTizen)
 
-
 [Learn more &gt;](../vstools/index.md)
```

Comment:
For more information, [see](../vstools/index.md)

Reply:
done